runtime/dolphin/src/lib.rs

```diff
@@ -129,7 +130,7 @@ mod weights;
 pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("dolphin"),
     impl_name: create_runtime_str!("dolphin"),
-    authoring_version: 1,
+    authoring_version: 2,
```

Comment:
Not sure this filed should be bumped.

Reply:
It must, because this release is incompatible with the current block production mechanism.

We might bump it in the release PR, but I think it doesn't hurt to add it here